tests/torch/test_pattern_manager.py

```diff
@@ -74,6 +74,7 @@
 IGNORING_IGNORED_PATTERN_REASONS = {
     IgnoredPatternNames.FC_BN_HSWISH_ACTIVATION: "Not relevant for Torch.",
     IgnoredPatternNames.EQUAL_LOGICALNOT: "Not relevant for Torch.",
+    IgnoredPatternNames.ROPE: "Not relevant for Torch.",
```

Comment:
Could you explain why this pattern is not relevant for other backends?

Reply:
There are no examples of this pattern in other backends. I can not add this for ONNX/PT since I have no models with this structure from these frameworks.

Reply:
What models did you use for testing the RoPE pattern for OV backend? If source models are PyTorch models you can reuse them to specify RoPE pattern for ONNX/PT backends.

Reply:
Ok

Reply:
Added patterns with the corresponding tests as you asked. Need to properly review it, since I've had to create it without strong ONNX/PT research due to lack of time.

Reply:
On which models have you tested the pattern?